Bottom: c63040a0874e4d576af521fdc830803fe9782aa0
Top:    fb407fce18baecba15d62668868dbcfa77f667f6
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-08-18 10:16:21 -0400

Refresh of sgtl5000

---

diff --git a/arch/arm/boot/dts/sun7i-a20.dtsi b/arch/arm/boot/dts/sun7i-a20.dtsi
index 32a9ea1..63542e3 100644
--- a/arch/arm/boot/dts/sun7i-a20.dtsi
+++ b/arch/arm/boot/dts/sun7i-a20.dtsi
@@ -860,7 +860,6 @@
 			compatible = "allwinner,sun7i-a20-pwm";
 			reg = <0x01c20e00 0xc>;
 			clocks = <&osc24M>;
-			#clock-cells = <1>;
 			#pwm-cells = <3>;
 			clock-output-names = "pwm0", "pwm1";
 			status = "disabled";
